solaris/solaris11/generate_wazuh_packages.sh

```diff
@@ -104,7 +104,7 @@ compile() {
         gmake -j $THREADS TARGET=agent PREFIX=${install_path} USE_SELINUX=no
     fi
 
-    $SOURCE/install.sh
+    DISABLE_SHARED=yes .$SOURCE/install.sh
```

Comment:
Hi @alguacilaguamara, this is ok but the `libwazuhext.so` has already been built by the previous `gmake` line. By default, this `DISABLE_SHARED` is set to `no`, so the shared object is always built by the Makefile.

In addition, if you change this, you should also change the "spec" file located in https://github.com/wazuh/wazuh-packages/blob/master/solaris/solaris11/SPECS/. You can do this change for all Wazuh's version (which I don't recommend) or add a check to set `DISABLE_SHARED` to `yes` only if you are building a version equal or greater to v3.10.0.

Reply:
In addition, please add some tests to this PR.